packages/docusaurus-theme-classic/src/theme/NavbarItem/DropdownNavbarItem.tsx

```diff
@@ -78,28 +91,27 @@ function DropdownNavbarItemDesktop({
   }, [dropdownRef]);
 
   return (
+    // eslint-disable-next-line jsx-a11y/no-static-element-interactions
     <div
       ref={dropdownRef}
-      className={clsx('navbar__item', 'dropdown', 'dropdown--hoverable', {
+      onMouseEnter={() => setShowDropdown(true)}
+      onMouseLeave={() => setShowDropdown(false)}
+      onKeyDown={onKeyDown}
+      className={clsx('navbar__item', 'dropdown', {
         'dropdown--right': position === 'right',
         'dropdown--show': showDropdown,
+        // When hydrated, handle hover state in JS
+        // It need to be kept in sync with the button onClick
+        'dropdown--hoverable': !isBrowser,
       })}>
```

Comment:
It should be the same behavior as https://squareup.com navbar and https://www.radix-ui.com/primitives/docs/components/navigation-menu 

The `dropdown` can be collapsed with a **click** on the `button` even if it was open with a **hover**